index.js

```diff
@@ -57,7 +60,7 @@ module.exports = function (options, logger) {
 					cb();
 				}
 			})
-			.create();
+			.create({}, options);
```

Comment:
I'm not really sure what this does, since I don't know jack about Stampit. It looks like the only potential option left is `config`, so is this letting Stylint go get the config file? At this point, do the rules in the config file override the rules in `rules` or do the `rules` override the contents of `options.config`?